mParticle-Kochava.podspec

```
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
     s.name             = "mParticle-Kochava"
-    s.version          = "8.2.0"
+    s.version          = "9.0.0"
```

```suggestion
    s.version          = "8.2.0"
```
reverted version number change